include/sys/mount.h

```diff
@@ -16,7 +16,26 @@
 #ifndef _LIBPHOENIX_SYS_MOUNT_H_
 #define _LIBPHOENIX_SYS_MOUNT_H_
 
-extern int mount(const char *source, const char *target, const char *fstype, long mode, char *data);
+
+#define MS_MGC_VAL     0xc0ed0000u
+#define MS_RDONLY      1 << 0   /* Mount read-only.  */
+#define MS_NOSUID      1 << 1   /* Ignore suid and sgid bits.  */
+#define MS_NODEV       1 << 2   /* Disallow access to device special files.  */
+#define MS_NOEXEC      1 << 3   /* Disallow program execution.  */
+#define MS_SYNCHRONOUS 1 << 4   /* Writes are synced at once.  */
+#define MS_REMOUNT     1 << 5   /* Alter flags of a mounted FS.  */
+#define MS_MANDLOCK    1 << 6   /* Allow mandatory locks on an FS.  */
+#define MS_DIRSYNC     1 << 7   /* Directory modifications are synchronous.  */
+#define MS_NOSYMFOLLOW 1 << 8   /* Do not follow symlinks.  */
+#define MS_NOATIME     1 << 10   /* Do not update access times.  */
+#define MS_NODIRATIME  1 << 11  /* Do not update directory access times.  */
```

Comment:
please write the values in parentheses, eg `(1 << 1)` as otherwise writing eg. `MS_NODEV + MS_NOEXEC` will be resolved to: `1 << 2 + 1 <<  3`   - which (due to c operators precedence) will result in unexpected value 64 (`+` is being more important than `<<`) instead of expected 12.

Hopefully nobody would use `+` in this context (as `|` should be used) but it's always good to use parentheses for non-trivial macro values.